packages/nimble-components/src/label-provider/core/label-tokens.ts

```diff
@@ -0,0 +1,17 @@
+import { DesignToken } from '@microsoft/fast-foundation';
+import { coreLabelDefaults } from './label-token-defaults';
+
+export const alertDismissLabel = DesignToken.create<string>({
+    name: 'alert-dismiss-label',
+    cssCustomPropertyName: null
+}).withDefault(coreLabelDefaults.alertDismissLabel);
+
+export const numberFieldDecrementLabel = DesignToken.create<string>({
+    name: 'number-field-decrement-label',
```

Comment:
Can we do numeric-increment and numeric-decrement?
Feel like anything numerical that may have those operations could use it, doesn't have to be specifically number-field (though is today)

Reply:
Updated